src/plugins/intel_cpu/src/nodes/executors/fullyconnected_implementations.cpp

```diff
@@ -225,6 +230,35 @@ OV_CPU_MAYBE_UNUSED_FUNCTION static inline bool noPostOps(const FCConfig& config
 template <>
 const std::vector<ExecutorImplementation<FCAttrs>>& getImplementations() {
     static const std::vector<ExecutorImplementation<FCAttrs>> fullyconnectedImplementations {
+#if defined(OV_CPU_WITH_KLEIDIAI)
+        OV_CPU_INSTANCE_KLEIDIAI(
```

Comment:
We shouldn't enable it with highest priority. I don't expect it to be faster than ACL in any scenario. 
So I would propose to put it below ACL executor. In future PRs we will extent its coverage with Weight Compression cases,

Reply:
Moved below acl executors